src/components/DelegationDashboard/DelegateCard.tsx

```diff
@@ -88,10 +105,23 @@ const DelegateCard = ({ delegate, className, trackNum, disabled }: Props) => {
 					)}
 
 					{!!delegate?.dataSource?.includes(EDelegateSource.W3F) && (
-						<div className='flex items-center space-x-3'>
+						<div className='flex items-center space-x-2 sm:space-x-3'>
 							<div className='flex items-center space-x-1'>
-								<W3FIcon />
-								<span className='text-xs font-normal text-bodyBlue dark:text-blue-dark-high'>W3f</span>
+								<Image
+									src={'/assets/profile/w3f.svg'}
```

Comment:
just use isMobile keyword to check display instead of hidden classaname and two times initialization of <Image